main_cgi.cpp

```diff
@@ -96,7 +96,12 @@ int try_main(int argc, char* argv[])
 
     if(argc == 2 && argv[1] == std::string("--capture"))
         {
-        std::system("set > settings");
+        int exit_code = std::system("set > settings");
```

Comment:
We probably should run `env` instead of `set` under Unix. But I'm not really sure what is this supposed to do.

Reply:
The description below the code says:
```c++
        std::cout
            << "Environment and QUERY_STRING captured for later use\n"
            << "in debugging. File 'settings' can be turned into a\n"
            << "command script in an OS-dependent way. After executing\n"
            << "the script, run the program, redirected its input from\n"
            << "file 'stdin.txt'."
            ;
```

Reply:
So this should definitely be `/usr/bin/env` under Unix.

Reply:
Fixed.
Tested `opt/lmi/bin/antediluvian_cgi --capture` and `settings` with environment variables appears.